### DISCUSSION TITLE
🧩 FEATURE REQUEST:  Able to share friend link of medium

### DISCUSSION THREAD

**Discussion by wagenrace** [dailydotdev/daily#1019] [Feature Requests]
### Concise Description

For authors to be able to share there no paywall (friend link) of medium articles

### Description

I write on medium but have a paywall there.
However, I want to share my friendlink (link without a paywall) on daily.dev, this would break the "no self-promotion" rule. 

### Solution Proposed

the check for medium links might need to become a bit more intelligent. Not only "Does this have a paywall" but "If it has a paywall do I have the friend link or not",
The rule of self-promotion may be ignored if the article is in a publication that is accepted as a source. 

### Code of Conduct

- [X] I follow the conditions of this project.

**Comment by idoshamun** [comment]
Hi @wagenrace,

We've got several people asking for a friend link support. We'll see how we can address that.
May take a while 🙏
